GDJS/GDJS/IDE/ExporterHelper.cpp

```diff
@@ -85,6 +85,8 @@ bool ExporterHelper::ExportLayoutForPixiPreview(gd::Project & project, gd::Layou
 
     //Export resources (*before* generating events as some resources filenames may be updated)
     ExportResources(fs, exportedProject, exportDir);
+
+    AddMainLibsInclude(true, false, includesFiles);
```

Comment:
I'm wondering, maybe we could simply move the call to `AddLibsInclude` here, where you added `AddMainLibsInclude`, **before** ExportEventsCode?
If it works we would not need an extra "AddMainLibsInclude"

Reply:
I tried that but `AddLibsInclude` includes other `gdjs` renderers that needs libs included by `ExportResources` as `gdjs` itself :/

Reply:
Mmmm I see... Maybe in this case we should move all the GDJS related includes (From line `InsertUnique(includesFiles, "libs/jshashtable.js");` to line `InsertUnique(includesFiles, "events-tools/networktools.js");`) inside `AddLibsInclude` (so that we have only one function that add the needed includes!)

It would be simpler (just one function, easy to reorder), and the function called `ExportEventsCode` would only export events code (as the name implies ;)).

Can you try this? To move all the InsertUnique (and the two comments) from ExportEventsCode to AddLibsInclude and then call `AddLibsInclude` before ExportEventsCode.

Quite sure it could work!

Also you have to update `Exporter::ExportWholeCocos2dProject` and `Exporter::ExportWholePixiProject` (I think you forget to do this also in your current solution - that's another reason to avoid adding an extra function ;)).